int32 id
string class_name
int32 bbox_x1
int32 bbox_y1
int32 bbox_x2
int32 bbox_y2
float32 confidence
float32 speed
uint8[] segmentation_mask